tests/dockerfile/checks/test_UserExists.py

```diff
@@ -15,14 +15,20 @@ def test(self):
         report = runner.run(root_folder=test_files_dir, runner_filter=RunnerFilter(checks=[check.id]))
         summary = report.get_summary()
 
-        passing_resources = {"/success/Dockerfile.USER"}
-        failing_resources = {"/failure/Dockerfile."}
+        passing_resources = {
+            "/success_user/Dockerfile.",
+            "/success_gosu/Dockerfile."
+        }
+        failing_resources = {
+            "/failure_no_user/Dockerfile.FROM",
+            "/failure_no_gosu/Dockerfile.FROM",
```

Comment:
can you also add a test case for the `gosu` command being inside the `CMD`?

Reply:
done